packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -309,10 +320,12 @@ class _InkResponseState<T extends InkResponse> extends State<T> {
       widget.onDoubleTap();
   }
 
-  void _handleLongPress() {
+  void _handleLongPress(BuildContext context) {
     _currentSplash?.confirm();
     _currentSplash = null;
     if (widget.onLongPress != null)
+      if (widget.enableFeedback)
+        Feedback.forLongPress(context);
       widget.onLongPress();
```

Comment:
Oh! Good catch!

I still wish `if`s would always come with braces... [goto fail;](https://www.synopsys.com/blogs/software-security/understanding-apple-goto-fail-vulnerability-2/) :wink:

As to writing a test: there is actually another `onLongPress != null` check that guards the entire execution of this method (see line 359). So, it's actually impossible to exploit the missing braces in a test here.